### DISCUSSION TITLE
[Feature] iOS - Merge people

### DISCUSSION THREAD

**Discussion by warnerbryce** [immich-app/immich#9819] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

For the moment on the iOS app we can’t merge same people in the search tab. We can edit name that’s all. 
For someone (my wife) who never use a computer it’s quite lame to not be able to merge people from iOS interface.
I don't have access to an android phone to test, but as you are using Flutter for both apps, it might be also the case. 

A part from that, she really enjoys this app and think it’s way better than Nextcloud for backing up photos. 

Speaking of which Nextcloud just updated today it iOS App and navigation in the Media Tab is way better than before, really more reactive… but still not compare with Immich.
Keep the great work ! You have a new supporter !

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6259

**Reply by warnerbryce** [reply]
Sorry, i put iOS in the search, so this Discussion didn't show up. 
We can close mine ?
